forum/models/model_utils.py

```diff
@@ -634,7 +659,7 @@ def handle_threads_query(
                     (page - 1) * per_page
                 ).limit(per_page)
                 threads = list(paginated_collection)
-                num_pages = max(1, (thread_count // per_page))
+                num_pages = max(1, math.ceil(thread_count / per_page))
```

Comment:
nit: The `max(...)` part is no longer necessary here. It used to be, because the `//` division could result in a zero result, but this is no longer the case here with `weil`.